platforms/bukkit/common/src/main/java/com/dfsek/terra/bukkit/util/BukkitUtils.java

```diff
@@ -0,0 +1,12 @@
+package com.dfsek.terra.bukkit.util;
+
+import org.bukkit.Material;
+import org.bukkit.block.data.BlockData;
+
+
+public class BukkitUtils {
+    public static boolean isLiquid(BlockData blockState) {
```

Comment:
This is only used once. Does this really warrant a whole utils class?

Reply:
it is a useful function to have as fabric, the reference platform has an easy way to do this, while bukkit does not. this is here so that it's easier to have development parity between fabric and bukkit